shared/teams/team/nav-header/index.new.tsx

```diff
@@ -154,6 +181,41 @@ const _HeaderTitle = (props: HeaderTitleProps) => {
     </Kb.Box2>
   )
 
+  const addInviteAndLinkBox = (
+    <Kb.Box2
+      direction="vertical"
+      gap={Styles.isMobile ? 'xtiny' : 'tiny'}
+      style={styles.addInviteAndLinkBox}
+      alignItems="center"
+      alignSelf="flex-end"
+    >
+      <AddPeopleButton teamID={props.teamID} />
+      {flags.teamInvites && (
+        <Kb.Text type={Styles.isMobile ? 'BodyTiny' : 'BodySmall'}>
+          {Styles.isMobile ? 'or' : 'or share a link:'}
+        </Kb.Text>
+      )}
+      {flags.teamInvites &&
+        (Styles.isMobile ? (
+          <Kb.Button
+            label="Generate invite link"
+            onClick={props.onManageInvites}
+            style={Styles.globalStyles.flexGrow}
+            mode="Secondary"
+            fullWidth={true}
+          />
+        ) : (
+          <Kb.Box2 direction="vertical" gap="xtiny" alignItems="flex-start">
+            <Kb.CopyText text="https://keybase.io/team/link/blahblah/" />
+            <Kb.Text type="BodyTiny">Adds as writer • Expires 10,000 ys</Kb.Text>
+            <Kb.Text type="BodyTiny" onClick={props.onManageInvites} className="hover-underline">
+              Manage invite links
+            </Kb.Text>
+          </Kb.Box2>
+        ))}
```

Comment:
Since these are siblings why not just wrap in one `flags.teamInvites && ( ... )`?

Reply:
@jakob223 Pointed out that `gap` doesn't work with a Fragment, which is correct. So we would need to update how `Box2` handles `gap` to traverse into Fragments and insert the spacers between.